apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -647,37 +654,38 @@ function constructInvestigationRulingPdf(
     })
   }
 
-  doc
-    .text(' ')
-    .text(
-      `${formatAppeal(existingCase.prosecutorAppealDecision, 'Sækjandi')} ${
-        existingCase.prosecutorAppealDecision === CaseAppealDecision.APPEAL
-          ? existingCase.prosecutorAppealAnnouncement ?? ''
-          : ''
-      }`,
-      {
-        align: 'justify',
-        paragraphGap: 1,
-      },
-    )
+  let prosecutorAppeal = formatAppeal(
+    existingCase.prosecutorAppealDecision,
+    'Sækjandi',
+  )
 
-  // Only show accused appeal decision if applicable
-  if (
-    existingCase.accusedAppealDecision !== CaseAppealDecision.NOT_APPLICABLE
-  ) {
-    doc
-      .text(' ')
-      .text(
-        `${formatAppeal(existingCase.accusedAppealDecision, 'Varnaraðili')} ${
-          existingCase.accusedAppealDecision === CaseAppealDecision.APPEAL
-            ? existingCase.accusedAppealAnnouncement ?? ''
-            : ''
-        }`,
-        {
-          align: 'justify',
-          paragraphGap: 1,
-        },
-      )
+  if (prosecutorAppeal) {
+    prosecutorAppeal = `${prosecutorAppeal} ${
+      existingCase.prosecutorAppealAnnouncement ?? ''
+    }`
+  } else {
+    prosecutorAppeal = existingCase.prosecutorAppealAnnouncement ?? ''
+  }
+
+  if (prosecutorAppeal) {
+    doc.text(' ').text(prosecutorAppeal, { align: 'justify', paragraphGap: 1 })
+  }
+
+  let accusedAppeal = formatAppeal(
+    existingCase.accusedAppealDecision,
+    'Varnaraðili',
```

Comment:
Wondering if 'Varnaraðili' as well as the strings in formatAppeal could be added to CF?

Reply:
They could, but I decided not to add them at this time. Added so many others on the client side. I would like to leave this for another day.